### PR TITLE
Add detailed Extended Key Usage output

### PR DESCRIPTION
**Pull request by dadrian**
DO NOT MERGE.

### CODE REVIEW

**Comment by dadrian** [comment]
@zakird, can you review, but don't click merge even if you approve?

**Comment by zakird** [comment]
This looks fine to me.

**Comment by dadrian** [comment]
@zakird Can you confirm the new changes?

File: x509/root_darwin.go [hunk 1]
```diff
@@ -2,7 +2,7 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
-//go:generate go run root_darwin_arm_gen.go -output root_darwin_armx.go
+// DISBLED: go:generate go run root_darwin_arm_gen.go -output root_darwin_armx.go
```
Comment (zakird) [inline x509/root_darwin.go]: DISBLED?

Comment (dadrian) [inline x509/root_darwin.go]: FXED.

**Comment by dadrian** [comment]
@zakird awk?

**Pull request merged**